demo/app/main-page.ts

```diff
@@ -1,10 +1,10 @@
-import { EventData } from 'tns-core-modules/data/observable';
-import { Label } from 'tns-core-modules/ui/label';
-import { Page } from 'tns-core-modules/ui/page';
-import * as frameModule from "tns-core-modules/ui/frame";
+import { EventData } from '@nativescript/core';
+import { Label } from '@nativescript/core';
+import { Page } from '@nativescript/core';
```

Comment:
Probably would be nicer if you group all @nativescript/core imports into one.

Reply:
I agree, it will be updated.